circleguard/result.py

```diff
@@ -106,14 +107,16 @@ class RelaxResult(InvestigationResult):
     replay: :class:`~circleguard.loadable.Replay`
         The replay investigated.
     ur: int
-        The unstable rate of the replay. More information on UR available at
-        https://osu.ppy.sh/help/wiki/Accuracy#accuracy
+        The (unconverted) unstable rate of the replay. More information on UR
+        available athttps://osu.ppy.sh/help/wiki/Accuracy#accuracy
     ischeat: bool
         Whether the replay is cheated or not.
     """
     def __init__(self, replay: Replay, ur: int, ischeat: bool):
         super().__init__(replay, ischeat, ResultType.RELAX)
-        self.ur = ur
+        self.ur = ur_to_cv_ur(ur, replay.mods)
```

Comment:
imo `self.ur` should be what osu! returns, which is unconverted.

Reply:
I had that originally and then switched it, reasoning being we're comparing against cv ur to determine `ischeat`, so we should return cv ur as `ur` and keep ucv ur as the odd one out. 

It would be confusing to have `result.ur` return something higher than your threshold, and `result.ischeat` return `True`.

Reply:
ok makes sense. how about renaming `ucv_ur` to either `raw_ur` or `uncv_ur`?

Reply:
There doesn't seem to be a standard naming convention for unconverted ur (which means we get to make it up ourselves!) - I like `ucv_ur`, `raw` doesn't fit with the existing scheme (`cvUR`) and prepending `un` instead of `u` feels too vernose